src/style-property-map-readonly.js

```diff
@@ -29,7 +29,7 @@
     }
 
     var parsed = StyleValue.parse(property, propertyString);
-    if (typeof parsed == 'Array') {
+    if (parsed.constructor == Array) {
```

Comment:
When looking through JavaScript resources this was suggested as a more common way of doing this. This is no big deal either way and I will change it if you would like me to.